crates/dkg-core/src/primitives/states.rs

```diff
@@ -785,7 +792,7 @@ pub mod tests {
         let dkgs: Vec<_> = dkgs
             .into_iter()
             .map(|dkg| {
-                let (ndkg, bundle_o) = dkg.process_shares(&all_shares).unwrap();
+                let (ndkg, bundle_o) = dkg.process_shares(&all_shares, false).unwrap();
```

Comment:
Should we also have a test with `true` and verify we got X responses and Y complaints?